.github/actions/gh-cache-cleanup-on-merge/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "GH cache cleanup"
+description: "Performs the cleanup of cache entries related with already closed PR"
```

Comment:
source: https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#force-deleting-cache-entries

Reply:
Please document this new action in the README, with a sample usage workflow

Reply:
Done